### ISSUE TITLE
Is there a way to test this package using jenkins.plone.org?

### ISSUE THREAD

**Issue by rafaelbco** [plone/plone.app.versioningbehavior#24]
I've read some [documentation](http://docs.plone.org/develop/coredev/docs/intro.html) but I'm still lost. 

I'm trying to fix a bug, but I'm not working on buildout.coredev. I checked out plone.app.versioningbehavior and I'm using the buildout of my own project. I can run tests locally, however I want to be sure my changes will work with all the supported Plone versions. I'm not sure how can I do this.

@mauritsvanrees can you help me?

**Comment by mauritsvanrees** [comment]
That documentation *should* get you up and running. Do you have a specific question? Where do you get stuck?

But when you have created a branch of this package and a pull request, you can run your own job on the Jenkins server to test coredev with your branch. You can login with your github account. This page should have all the information you need:
http://jenkinsploneorg.readthedocs.io/en/latest/run-pull-request-jobs.html

**Comment by gforcada** [comment]
Thanks @mauritsvanrees for pointing to the docs :-) I'm working on improving mr.roboto to make it even easier, hopefully in weeks time only.

**Comment by rafaelbco** [comment]
@mauritsvanrees , my doubt is: if I just make a direct commit on the default branch, how can I see the tests results?

**Comment by gforcada** [comment]
@rafaelbco you *should* never commit directly to master, as best practices and to allow peer-reviewing, please always create a pull request that can be easily tested with jenkins.plone.org as Maurits already mentioned.

Still, indeed, if you make a commit on master mr.roboto will make an automatic commit on buildout.coredev (for the right branch(es) needed) and that will be tested on jenkins.plone.org. Right now there is no notification back to github and we still do not notify to the user that broke a build. That's on the @plone/testing-team todo's list, we are happy to get contributions ;-)

**Comment by rafaelbco** [comment]
Thanks @gforcada ! I've already contributed to some Plone packages (this one included), but this time I'd like to understand all the process. The other times I just made the pull requests and let other people do the rest.

Now I guess I'm on the right track, check it out: https://github.com/plone/plone.app.versioningbehavior/pull/26

I started the build on Jenkins and I'm waiting to complete. 

I already tested locally and made sure it's working on Plone 5 and 4.3.
